doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -201,6 +201,7 @@ nRF Desktop
     The option can be enabled even when the Bluetooth controller is not enabled as part of the application that uses :ref:`nrf_desktop_ble_latency`.
   * The :ref:`nrf_desktop_usb_state` to use :c:func:`usb_hid_set_proto_code` function to set the HID Boot Interface protocol code.
     The ``CONFIG_USB_HID_BOOT_PROTOCOL`` Kconfig option was removed and dedicated API needs to be used instead.
+  * Disabled MCUboot's logs over RTT (:kconfig:option:`CONFIG_LOG_BACKEND_RTT` and :kconfig:option:`CONFIG_USE_SEGGER_RTT`) on ``nrf52840dk_nrf52840`` in :file:`prj_mcuboot_qspi.conf` configuration to reduce MCUboot memory footprint and avoid flash overflows.
```

Comment:
Consider mentioning that UART backend has been enabled instead

Reply:
The UART backend was enabled all of the time (board enables it) :)

Reply:
Then why it has been enabled explicitly in configuration if it is not necessary in theory as board enables it?

Reply:
Ok, updated the release note :)

Reply:
Resolved